openapi/openapi.yaml

```diff
@@ -0,0 +1,299 @@
+              - 'Housing'
+              - 'Humanities'
+              - 'Information and Statistics'
+              - 'Income Security and Social Services'
+              - 'Law, Justice and Legal Services'
+              - 'Natural Resources'
+              - 'Other'
+              - 'Opportunity Zone Benefits'
+              - 'Regional Development'
+              - 'Science and Technology and Other Research and Development'
+              - 'Transportation'
+              - 'Affordable Care Act'
+          code:
+            type: string
+            enum:
+              - 'RA'
+              - 'AG'
+              - 'AR'
+              - 'BC'
+              - 'CD'
+              - 'CP'
+              - 'DPR'
+              - 'ED'
+              - 'ELT'
+              - 'EN'
+              - 'ENV'
+              - 'FN'
+              - 'HL'
+              - 'HO'
+              - 'HU'
+              - 'IS'
+              - 'ISS'
+              - 'LJL'
+              - 'NR'
+              - 'O'
+              - 'OZ'
+              - 'RD'
+              - 'ST'
+              - 'T'
+              - 'ACA'
+      FundingActivity:
+        type: object
+        properties:
+          categories:
+            type: array
+            items:
+              $ref: '#/components/schemas/FundingActivityCategory'
+            minItems: 1
+          explanation:
+            type: string
+            nullable: true
+      FundingInstrument:
+        type: object
+        properties:
+          name:
+            type: string
+            enum: [
+              'Cooperative Agreement',
+              'Grant',
+              'Procurement Contract',
+              'Other',
+            ]
+          code:
+            type: string
+            enum: [
+              'CA',
+              'G',
+              'PC',
+              'O',
+            ]
+      GrantorContact:
+        type: object
+        properties:
+          email:
+            $ref: '#/components/schemas/Email'
+          text:
+            type: string
+      Metadata:
+        type: object
+        properties:
+          version:
+            type: string
+      OpportunityCategory:
+        type: object
+        properties:
+          name:
+            type: string
+            enum:
+              - 'Discretionary'
+              - 'Mandatory'
+              - 'Continuation'
+              - 'Earmark'
+              - 'Other'
+          code:
+            type: string
+            enum:
+              - 'D'
+              - 'M'
+              - 'C'
+              - 'E'
+              - 'O'
+          explanation:
+            type: string
+      Opportunity:
+        type: object
+        properties:
+          id:
+            type: string
+          number:
+            type: string
+          title:
+            type: string
+          category:
+            $ref: '#/components/schemas/OpportunityCategory'
+          required:
+            - id
+            - number
+            - title
```

Comment:
I adjusted this in the previous comment's suggestion, but just to note here that it looks like this `required` block is over-indented by one level.

Reply:
it looks right now, can you confirm? I think with your suggestion it got fixed

Reply:
The indent on `requirements for the `Opportunity` model looks good.

However, it seems like something got messed up when applying the diff from my suggestion; there's now an incorrectly-indented `required` block on the `OpportunityMilestones` schema. Additionally, there's an errant `OpportunityMilestones.category` property.

I actually hadn't noted any fields as required in that suggestion, but looking at it now, it seems like those should be added. I'll add a final tweak in a separate comment that will (hopefully) add those and remove the `category` property.